File: app/src/main/java/com/ninpou/qbits/request/ResponseInfo.java
package com.ninpou.qbits.request;

import java.io.Serializable;
import java.util.Objects;

import okhttp3.Response;

public class ResponseInfo implements Serializable {
    private String content;
    private String header;

    public ResponseInfo(Response response) {
        header = response.headers().toString();
        try {
            content = Objects.requireNonNull(response.body()).string();
        } catch (Exception e) {
            content = "null";
        }
    }

    public String getHeader() {
        return header;
    }

    public String getContent() {
        return content;
    }
}

File: app/src/main/java/com/ninpou/qbits/request/ResponseInfo.kt
package com.ninpou.qbits.request

import okhttp3.Response
import java.io.Serializable
import java.util.*

class ResponseInfo(response: Response) : Serializable {
    var content: String? = null
    val header: String = response.headers().toString()

    init {
        content = try {
            response.body()?.string()
        } catch (e: Exception) {
            "null"
        }
    }
}